kolibri/core/auth/api.py

```
@@ -516,6 +517,25 @@
+        """
+        user_id = request.data.get("user_id", "")
+        try:
+            user = FacilityUser.objects.get(id=user_id)
+            if not request.user.can_delete(user):
+                raise PermissionDenied()
+
+            user.delete_imported_user()
+
+            return Response({"user_id": user_id})
+        except FacilityUser.DoesNotExist:
+            raise Http404("User does not exist")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)